src/Payments.sol

```diff
@@ -70,6 +70,7 @@ contract Payments is
         uint256 lockupAllowance;
         uint256 rateUsage; // Track actual usage for rate
         uint256 lockupUsage; // Track actual usage for lockup
+        uint256 oneTimePaymentsUsed; // Track one-time payments made by this operator
```

Comment:
Any reason not to reduce the allowance instead?

Reply:
I.e., reduce the usage _and_ allowance at the same time (clipped at 0)?

Reply:
Yeah that's a better idea. Closing this.